web/src/pages/Route/List.tsx

```diff
@@ -387,6 +387,9 @@ const Page: React.FC = () => {
             }}>
               {formatMessage({ id: 'component.global.view' })}
             </Button>
+            <Button type="primary" onClick={() => history.push(`/routes/${record.id}/duplicate`)}>
```

Comment:
Could we fetch data first, then submit it directly by using `create` method?

Reply:
we can't do that because the route name is unique, we need to take user to `/routes/:rid/duplicate` and fill the copied data, user needs to change the name and then get next to submit

Reply:
ok, got it